templates/admin/config.tmpl

```diff
@@ -152,7 +152,6 @@
 							<dt>{{.i18n.Tr "admin.config.mailer_name"}}</dt>
 							<dd>{{.Mailer.Name}}</dd>
 							<dt>{{.i18n.Tr "admin.config.mailer_disable_helo"}}</dt>
-							<dd><i class="fa fa{{if .Mailer.DisableHelo}}-check{{end}}-square-o"></i></dd>
```

Comment:
As @alexcesaro pointed out in go-gomail/gomail#52, the smtp package always sends a HELO (with "localhost" by default), so the setting DisableHelo is useless.